front-end/src/components/CreateEditPostModal.tsx

```diff
@@ -103,8 +104,17 @@ export default function CreateEditPostModal(props: Props){
 
       if(isCreate){
         axios.post(process.env.REACT_APP_API_URL + "/api/author/" + props.loggedInUser.authorId + "/posts/", data, config)
-          .then(res => {
-            handleRes(res)
+          .then(post => {
+            handleRes(post)
+            return post
+          }).then(post => {
+            // send this post to all followers
+            axios.get(`${process.env.REACT_APP_API_URL}/api/author/${props.loggedInUser.authorId}/followers/`).then(res => {
+              let followingList: Author[] = res.data.items;
+              followingList.forEach(follower => {
+                axios.post(`${process.env.REACT_APP_API_URL}/api/author/${follower.id}/inbox/`, post.data);
+              });
+            });
```

Comment:
On second thought, you can just get rid of this 2nd .then and put this code in place of "return post" above